.github/ISSUE_TEMPLATE/BUG_REPORT.yml

```diff
@@ -0,0 +1,102 @@
+name: "Bug report: Netdata"
+description: "Submit a report and help us improve our free and open-source Netdata Agent"
+title: "[Bug]: "
+labels: ["bug", "needs triage"]
+body:
+  - type: markdown
+    attributes:
+      value: "### Thank you for contributing to our project!"
+  - type: markdown
+    attributes:
+      value: |
+        <img src="https://img.shields.io/github/v/release/netdata/netdata.svg?label=latest%20stable"> <img src="https://img.shields.io/badge/dynamic/xml?url=https://storage.googleapis.com/netdata-nightlies/latest-version.txt&label=latest%20nightly&query=/text()">
+  - type: markdown
+    attributes:
+      value: |
+        Before submitting, we'd appreciate it if you:
+          - Verify that your issue is not already reported on GitHub.
+          - Check if your Netdata Agent is up to date. If not, we recommend that you [update](https://learn.netdata.cloud/docs/agent/packaging/installer/update) first.
+  - type: textarea
+    id: report-summary
+    attributes:
+      label: Bug report summary
+      description: Provide a clear and concise description of the bug you're experiencing.
+    validations:
+      required: true
```

Comment:
Should we maybe move the `Bug report summary` down, before the `Steps to reproduce` ?

Reply:
That is a good question in general about the order and I thought about it myself.
This PR keeps the order we have in [bug_report.md](https://github.com/netdata/netdata/blob/master/.github/ISSUE_TEMPLATE/bug_report.md), but I tend to agree that we might change it.

Let's think. We have:
 - Bug report summary
 - System info
 - Netdata build info
 - Installation method
 - Steps to reproduce
 - Expected behavior
 - Additional info

I like having "Bug report summary" on top, because it is the most important part for me, so there is no need to scroll to find it when opening an issue.

On the other hand, "Bug report summary" in fact is "Actual behavior", and having it near "Expected behavior" and "Steps to reproduce" makes sense too.

Could be:
 - Bug report summary (Actual behavior)
 - Expected behavior
 - Steps to reproduce
 - System info
 - Installation method
 - Netdata build info
 - Additional info

Reply:
> 
    Bug report summary (Actual behavior)
    Expected behavior
    Steps to reproduce
    System info
    Installation method
    Netdata build info
    Additional info

I think I agree with this... 🤔